themes/digital.gov/layouts/events/card-event.html

```diff
@@ -1,37 +1,44 @@
 {{ $path := "" }}{{ with .File }}{{ $path = .Path }}{{ else }}{{ $path = .Path }}{{ end }}
-<div class="card card-event-promo card-linked" data-edit-this="{{- $path -}}" {{ if .Params.short_url }}data-short_url="{{- .Params.short_url -}}"{{ end }}>
-  <div class="grid-row tablet:grid-gap-4">
 
-    <div class="grid-col-9 tablet-lg:grid-col-10">
-      {{- if .Params.kicker -}}
-      <div class="kicker">{{- .Params.kicker -}}</div>
-      {{- end -}}
-      <h3>
-        <a href="{{ .Permalink }}" title="{{ .Title | markdownify }}" rel="bookmark">{{ .Title | markdownify }}</a>
-      </h3>
-
-      {{- if .Params.summary -}}
-      <div class="summary">{{ .Params.summary | markdownify }}</div>
-      {{- else -}}
-      <div class="deck">{{ .Params.deck | markdownify }}</div>
-      {{- end -}}
-
-      {{- if .Params.host -}}
-      <p class="host">Hosted by Digital.gov and the {{ .Params.host }}</p>
-      {{- end -}}
-
-      <a [attr.aria-label]="Register for {{- .Params.title -}}" class="btn btn-register" href="{{- .Params.registration_url -}}" onclick="__gaTracker('send', 'event', 'outbound-article', '{{- .Params.registration_url -}}', 'REGISTER NOW');">Register</a>
-    </div>
-
-    <div class="grid-col-3 tablet-lg:grid-col-2 order-first">
-
-      <div class="date">
-        <span class="date-mo">{{- dateFormat "Jan" .Date -}}</span>
-        <span class="date-day">{{- dateFormat "02" .Date -}}</span>
-        <a [attr.aria-label]="Register for {{- .Params.title -}}" class="btn btn-register" href="{{- .Params.registration_url -}}" onclick="__gaTracker('send', 'event', 'outbound-article', '{{- .Params.registration_url -}}', 'REGISTER NOW');">Register</a>
-      </div>
+<article class="card card-event card-event-promo card-linked" data-edit-this="{{- $path -}}" {{ if .Params.short_url }}data-short_url="{{- .Params.short_url -}}"{{ end }}>
```

Comment:
Added shared card class `card-event` so we can re-use styles.

Reply:
**TODO**

We should look into creating a **single** card event component that handles both past and future events.

Reply:
Created issue for this TODO - https://github.com/GSA/digitalgov.gov/issues/5526.